participants/PinataSnapshots.md

```diff
@@ -0,0 +1,65 @@
+# Pinata
+
+## (0) Project Name and Team Slack Handles
+
+*Our project Name:* **Pinata Snapshots**
+
+*Team Slack Handles:* **Matt Ober**, **Justin Hunter**
+
+## (1) Brief Project Description
+
+"Snapshots" will provide Pinata users with a way to back up their pinsets to filecoin.
+
+## (2) Link to Project UI
+
+https://ipfs.io/ipfs/Qmebt4ZhKfZCxR3FVxnGtacwaioA72EVQ4Frzs5Wwhe7da
+
+## (3) What does your application/UI do?
+
+Pinata currently serves the decentralized storage ecosystem through our IPFS pinning service. 
+
+With "Snapshots", we're allowing our users to back up their IPFS pinsets in a decentralized, cost effective way.
+
+The "Snapshots" functionality will allow users to either create manual or scheduled snapshots of their pinsets that can be triggered either through our GUI interface (Documented API support likely coming soon).
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+N/A
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+The data backed up will be the complete pin sets of Pinata users who choose to back up their data.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+We plan to store a few Terabytes of data during the Slingshot competition
+
+## (7) How are you structuring the data?
+
+To create a snapshot we will shard users' pin sets into shards of less than 32GB in size (to fit sector limits) and create root dags for each shard. 
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+To create a snapshot we will shard users' pin sets into shards of less than 32GB in size (to fit sector limits) and create root dags for each shard. 
+
+## (9)  What tech stack will you use to store the data?
+
+The dags created will be stored on filecoin via the Lotus Filecoin client
```

Comment:
For retrieval, will you be using an IPFS node or something like Powergate?

Reply:
Since our content is stored on our IPFS nodes already (before it's backed up), the content will be easily retrievable either via an IPFS gateway or via an IPFS node.

For filecoin based retrievals, we'll be providing the deal / miner information associated with each snapshot. With this information, users will be able to retrieve data in whatever manner they see fit.